src/main/java/io/cdap/plugin/dlp/SensitiveRecordRedaction.java

```diff
@@ -375,8 +402,8 @@ public void validate(FailureCollector collector, Schema inputSchema) {
           // If the user has a custom template enabled but doesnt use it in any of the transforms
           if (!anyTransformUsedCustomTemplate && this.customTemplateEnabled) {
             collector.addFailure("Custom template is enabled but no transforms use a custom template.",
-                                 "Please define a transform that uses the custom template or disable the custom "
-                                   + "template.")
```

Comment:
Why are these changed ?

Reply:
Those shouldnt have changed, reverted it